docs/roman/tweakreg/tweakreg_examples.rst

```diff
@@ -34,7 +34,7 @@ or a Roman datamodel `ImageModel`.
                 step.save_abs_catalog = True # save the catalog data used for absolute astrometry
                 step.abs_refcat = 'GAIADR3' # use Gaia DR3 for absolute astrometry
                 step.catalog_path = '/path/for/the/abs/catalog' # save the Gaia catalog to this path
-                step.call([img])
```

Comment:
This use of `call` didn't use the `step` instance so the custom parameters were ignored. I changed the other uses in this file to `run` to match.